### ISSUE TITLE
How to extract floating point logits after segmentation?

### ISSUE THREAD

**Issue by harry-nguyen-1234** [DeepSceneSeg/EfficientPS#55]
Dear authors,

Thank you for your fascinating work!
I would like to perform transfer learning using the EfficientPS network.
Is it possible to extract the floating point logits of the semantic, instance and fused branches before they are converted into panoptic IDs? If so, could you please point me to where I could access this information?

Best regards.
